### PR TITLE
Add/example/with fingerprintjs

### PR DESCRIPTION
**Pull request by ilfa**
It is the PR for pre review before showing it to Vercel team.
Here is the instruction [how to add new example](https://github.com/vercel/next.js/blob/canary/contributing.md#adding-examples).

### CODE REVIEW

File: examples/with-fingerprintjs/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Example app with FingerprintJS Pro
```
Comment (makma) [inline examples/with-fingerprintjs/README.md:1]:
```suggestion
# An example app with FingerprintJS Pro
```

File: examples/with-fingerprintjs/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+pnpm create next-app --example with-fingerprintjs with-fingerprintjs-app
+```
+
+Deploy it to the cloud with [Vercel](https://vercel.com/new?utm_source=github&utm_medium=readme&utm_campaign=next-example) ([Documentation](https://nextjs.org/docs/deployment)).
+
+## Notes
+
+### API key
+
+In order to identify visitors, you'll need a FingerprintJS Pro account (you can [sign up for free](https://dashboard.fingerprintjs.com/signup/)).
```
Comment (makma) [inline examples/with-fingerprintjs/README.md:29]:
```suggestion
To identify visitors, you'll need a FingerprintJS Pro account (you can [sign up for free](https://dashboard.fingerprintjs.com/signup/)).
```

File: examples/with-fingerprintjs/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+You can learn more about API keys in the [official FingerprintJS Pro documentation](https://dev.fingerprintjs.com/docs/quick-start-guide).
+
+When you obtain your own public key, you need to set up API key in environment variable `NEXT_PUBLIC_FPJS_PUBLIC_API_KEY`.
```
Comment (makma) [inline examples/with-fingerprintjs/README.md:32]:
```suggestion
Once you get public key, you need to set up the API key in the environment variable `NEXT_PUBLIC_FPJS_PUBLIC_API_KEY`.
```

File: examples/with-fingerprintjs/components/Nav.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import { PropsWithChildren } from 'react'
+import { useRouter } from 'next/router'
+import Link from 'next/link'
+import { RouterProps } from './types'
+
+export const Nav: React.FC = () => {
+  const router = useRouter();
+  console.log(router);
```
Comment (TheUnderScorer) [inline examples/with-fingerprintjs/components/Nav.tsx:8]: Do we need to keep this console.log?

**Pull request closed without merging**